src/renderer/components/table/table-cell.tsx

```diff
@@ -63,14 +68,30 @@ export class TableCell extends React.Component<TableCellProps> {
   }
 
   render() {
-    const { className, checkbox, isChecked, sortBy, _sort, _sorting, _nowrap, children, title, renderBoolean: displayBoolean, ...cellProps } = this.props;
+    const { onResize, onResizeStop, className, isResizable, checkbox, isChecked, sortBy, _sort, _sorting, _nowrap, children, title, renderBoolean: displayBoolean, ...cellProps } = this.props;
     const classNames = cssNames("TableCell", className, {
       checkbox,
       nowrap: _nowrap,
       sorting: this.isSortable,
     });
     const content = displayBooleans(displayBoolean, title || children);
 
+    if (isResizable){
+      return (
+        <Resizable
+          as="div"
+          onClick={this.onClick}
+          className = {classNames}
+          enable={{top:false, right:true, bottom:false, left:false, topRight:false, bottomRight:false, bottomLeft:false, topLeft:false }}
```

Comment:
I cannot currently try this out as I get an error, but it seems strange that we can only resize to the right.

Reply:
IMHO, Adding one control element is enough to manage the border between headers